guides/source/testing.md

```diff
@@ -1431,7 +1430,7 @@ class UserMailerTest < ActionMailer::TestCase
     assert_equal ['me@example.com'], email.from
     assert_equal ['friend@example.com'], email.to
     assert_equal 'You have been invited by me@example.com', email.subject
-    assert_equal read_fixture('invite').join, email.body.to_s
+    assert_equal read_fixture('invite').join, email.text_part.body.to_s
```

Comment:
`to_s` just returns the decoded version when only one part is present, is the preferred way, again in case of a single part: 
https://github.com/mikel/mail/blob/4876ae4ff33cb3a564de0a27b819ddb7f5123bee/lib/mail/body.rb#L184

The `NOTE` below specifically mentions what to do in case of multipart.